packages/turbo-workspace-convert/src/install.ts

```diff
@@ -0,0 +1,41 @@
+import execa from "execa";
+import ora from "ora";
+import { satisfies } from "semver";
+import { PACKAGE_MANAGERS, InstallArgs } from "./types";
+
+async function install(args: InstallArgs) {
```

Comment:
I'm tempted to say that this should clean up `node_modules` first for any resolution algorithm path between workspace root and the workspace path. (e.g. `workspaceRoot/node_modules`, `workspaceRoot/packages/node_modules`, `workspaceRoot/packages/a/node_modules`)

Running an install on top of another package manager's install seems like a bad day waiting to happen.

Reply:
Yep node modules will be gone by the time we get here.